skedge-backend/middlewares/decrypt-requests.js

```diff
@@ -2,12 +2,12 @@ const decrypt = require("../web_api_utilities/rsa-encryption").decryptStringWith
 
 module.exports = function (req, res, next) {
     try {
-        let verbose_logging = false;
+        let verbose_logging = true;
```

Comment:
I like this - being able to toggle the big logs is good. However I also think this should be set to false on the master branch, and only set to true temporarily when debugging.

That being said, don't start adding these ifs everywhere or the code will get messy. Ideally we would start using a logging framework like winston.

Reply:
yeah... a logging framework would be nice. Was really just for this module. Will update the var